osu.Game/Graphics/UserInterface/FilteringSearchList.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using osu.Framework;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+
+namespace osu.Game.Graphics.UserInterface
+{
+    public class FilteringSearchList<T> : Container where T : Drawable

```

Comment:
where part should be on new line (do you not use formatting in visual studio? it will do this for you)